src/App.js

```diff
@@ -60,7 +25,7 @@ function App() {
                     <Route path='/create/:id' exact>
                         <Details />
                     </Route>
-                    <Route path='/embed/:id/:token1/:token2' exact>
+                    <Route path='/embed/:id/:baseToken/:quoteCurrencyToken' exact>
```

Comment:
is there a reason that the quote token is called `quoteCurrencyToken` rather than `quoteToken`?

Reply:
It was named as `quoteCurrencyToken`. It can be renamed on the future releases